### PR TITLE
Partnership page fe

### PR DESCRIPTION
**Pull request by axelsson**
First version out! An ugly yet functional solution. Have not yet figured out how to get the translations to work in staging so hopefully it just works in prod 🤞. The files (in public/i18n/..) should be generated as soon as you start the server, and automatically updated if you make any changes in the yml-translation files.

If you're wondering why I've touched so many unrelated translation files it's cause their naming (numbers) caused a problem for the lang exporter.

Please point out if I've done anything unconventional in regard to rails/tailwind/anything.

Testing
Test with an organization that you can edit in admin to make it a partner and add a key that will be used in the url. Make sure your user has the same org id set. By then you should be able to see "companyname dashboard" in the user menu.

127 0 0 1_3000_se_organization_goclimate
127 0 0 1_3000_organization_goclimate

### CODE REVIEW

**Comment by axelsson** [comment]
[Partnership pages (frontend)](https://trello.com/c/bXIiVMPf/811-partnership-pages-frontend)

**Review by nanditagoclimate — APPROVED** [review]
I have one comment but otherwise this PR looks good to me! Good job on this. 🎉

File: config/environments/test.rb [hunk 1]
```diff
@@ -50,4 +50,8 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+
+  # Generate translation files for frontend
```
[the remark this answers is not in the document]
Comment (axelsson) [reply config/environments/test.rb:54]: The initializer (config/initializers/i18n.rb) generates these files when I start the server locally so I'm hoping for it to work in prod as well. Although it didn't run in staging so I'm not too sure 😬

**Review by kallenilver — APPROVED** [review]
Looks good, lets try it out.

I've done some reading regarding if this should work on prod in Heroku or not. I know files generated will not persist between restarts, but since this (and other assets) are generated during init I don't see a clear reason why this would not work in prod. So, lets try :)

**Pull request merged**